src/firefly/js/visualize/saga/CoverageWatcher.js

```diff
@@ -769,17 +770,16 @@ function getBoxAryFromTable(options,table, usesRadians){
         .filter( (row) => row.every( (v) => v));
 }
 
-
-function getRegionAryFromTable(options, table) {
+const getRegionAryFromTable = memorizeLastCall( (table) => {
```

Comment:
@robyww ran into a bug after my own changes. Tried uploading a file without a s_region column. And the coverage map would not load, because on line 338 when I call `const regionAry = getRegionAryFromTable(table);`, in **getRegionAryFromTable** function, we try to access rCol.regionIdx but rCol is null (because` rCol = findTableRegionColumn(table);` and there is no region column!)...

so added a null check for `rCol` here and that fixes this issue. 

Also wrapped memorizeLastCall around **getRegionAryFromTable** as you suggested. Do these 2 changes look good? I'll go ahead and merge if no more issues..